content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -1,9 +1,79 @@
 # Setup a Google Drive DVC Remote
 
-Follow this guide to setup Google Drive as your <abbr>DVC project</abbr>'s
-[remote storage](/doc/command-reference/remote).
+In this guide we explain the existing ways to setup Google Drive
+[remote storage](/doc/command-reference/remote) for your <abbr>DVC
+projects</abbr>, along with the different benefits each one brings.
 
-## Configure a Google Cloud project
+Note that to start using a GDrive remote, you only need to add it with a valid
+URL (see next section), and use any DVC command that requires it (see
+`dvc pull`, `dvc fetch`, `dvc push`). For example:
+
+```dvc
+$ dvc remote add -d myremote gdrive://root/dvc/dvcstore
+$ dvc add file.dat
+$ dvc push -r myremote
+...
+Go to the following link in your browser:
+
+    https://accounts.google.com/o/oauth2/auth # ... copy this link
+
+Enter verification code: # <- enter resulting token
+```
+
+See the [Authorization](#authorization) section for more details.
+
+## URL format for Google Drive remotes
+
+A GDrive remote URLs is needed for the `dvc remote add` command. It can be
+constructed with a _base_, and an **optional** _path_ to a folder i.e.
+`gdrive://<base>/path/to/folder`. The base can be one of:
+
+1. `root` - indicates your topmost Google Drive directory.
+
+```dvc
+$ dvc remote add -d mygdroot gdrive://root
```

Comment:
let's not use empty `//root` as an example - we should probably raise an exception if it's used

Reply:
Shouldn't it be up to the user? If it's not a good practice then I'll use a different example ⏳ but not sure about the exception. Maybe a warning. Anyway, that's a core repo issue 🙂

Reply:
Removed line, and opened dvc/issues/3586

Reply:
thanks! a good one. Made me think once again about basic scenario